### PR TITLE
feat(pci.project.private-registry): change guides urls

### PR DESCRIPTION
**Pull request by Jisay**
# :house_with_garden: Internal
* MANAGER-4326

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html [hunk 1]
```diff
@@ -3,9 +3,20 @@
...
+            <oui-guide-menu-item
+                data-href="{{ $ctrl.GUIDE_LINKS.CREATING_AND_USING_A_PRIVATE_IMAGE }}"
+                data-external
+            >
+                <span
+                    data-translate="private_registry_guide_creating_and_using_a_private_image"
+                ></span>
+            </oui-guide-menu-item>
+            <oui-guide-menu-item
+                href="{{ $ctrl.GUIDE_LINKS.USING_PRIVATE_REGISTRY_WITH_KUBERNETES }}"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html]: Might be one of the only case where `data-` is mandatory 😛

File: packages/manager/modules/pci/src/projects/project/private-registry/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -61,5 +61,7 @@
   "private_registry_beta_warning": "Ce produit est actuellement en version bêta, vous devez accepter les termes de la version bêta pour continuer.",
   "private_registry_accept": "J'accepte",
   "private_registry_accept_beta_terms": "les termes de la bêta",
-  "private_registry_new_project_contracts": "J’ai lu et j’accepte les contrats : {{ contractsLinks }}."
+  "private_registry_new_project_contracts": "J’ai lu et j’accepte les contrats : {{ contractsLinks }}.",
+  "private_registry_guide_creating_and_using_a_private_image": "Création et utilisation d'images privées",
+  "private_registry_guide_using_private_registry_with_kubernetes": "Utilisation d'OVH Managed Private Registry avec OVH Managed Kubernetes"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/private-registry/translations/Messages_fr_FR.json]:
```suggestion
  "private_registry_guide_using_private_registry_with_kubernetes": "Utilisation d'OVHcloud Managed Private Registry avec OVHcloud Managed Kubernetes"
```

File: packages/manager/modules/pci/src/projects/project/private-registry/private-registry.constants.js [hunk 3]
```diff
@@ -37,14 +37,26 @@ export const GUIDES = [
   },
 ];
 export const REGION = 'GRA7';
-export const GUIDELINK = 'https://docs.ovh.com/';
+export const GUIDE_LINKS = [
+  {
+    key: 'CREATING_AND_USING_A_PRIVATE_IMAGE',
+    url:
+      'https://docs.ovh.com/gb/en/private-registry/creating-and-using-a-private-image/',
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/private-registry/private-registry.constants.js]: only english ?

File: packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html [hunk 4]
```diff
@@ -3,9 +3,12 @@
         <oui-guide-menu
             data-text="{{:: 'pci_project_guides_header' | translate }}"
         >
-            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+            <oui-guide-menu-item
+                data-ng-repeat="guide in $ctrl.GUIDE_LINKS track by $index"
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html]:
```suggestion
                data-ng-repeat="guide in $ctrl.GUIDE_LINKS track by guide.key"
```

File: packages/manager/modules/pci/src/projects/project/private-registry/private-registry.constants.js (hunk 3 above)
Comment (Jisay) [reply packages/manager/modules/pci/src/projects/project/private-registry/private-registry.constants.js]: yes

File: packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html [hunk 5]
```diff
@@ -3,9 +3,12 @@
...
+                data-href="{{ guide.url }}"
+            >
                 <span
-                    data-translate="pci_project_guides_header_all_guides"
+                    data-translate="{{ 'private_registry_guide_' + guide.key.toLowerCase() }}"
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html]:
```suggestion
                    data-translate="{{:: 'private_registry_guide_' + guide.key.toLowerCase() }}"
```

**Pull request merged**